umbraco/Umbraco-CMS
# v14: Remove mentions of UmbracoApiController

# Notes
- Removes `UmbracoApiController`, `UmbracoApiControllerBase` & `UmbracoApiControllerAttribute` classes, and mention of those.
- This is because this pattern is no longer used at all, and is instead replace with the `UmbracoManagementApiControllerBase` instead.

# How to test
- The solution should build, and the backoffice should work as is.
- Create a document type named "Root" with allow as root set to true.
- Create a template also called "Root"
- Go to your document type, and be sure to select the template you just created.
- Create some content with the document type, again ensure the template on the content is correct.
- Go to the `/` route and assert you content is rendered 👍
